pkg/apis/radix/v1/radixapptypes.go

```diff
@@ -936,7 +936,7 @@ type RadixVolumeMount struct {
 	Container string `json:"container,omitempty"` // Outdated. Use Storage instead
 
 	// Storage defines the name of the container in the external storage resource.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
```

Comment:
```suggestion
	// Deprecated: use BlobFuse2 instead.
	// Storage defines the name of the container in the external storage resource.
	// +optional
```
The first comment must start with "Deprecated: " for field to be flagged as deprecated

Reply:
Fixed